crates/forge/tests/e2e/fuzzing.rs

```diff
@@ -91,7 +91,7 @@ fn fuzzing_set_seed() {
         [PASS] fuzzing::tests::fuzzed_argument (fuzzer runs = 256)
         [PASS] fuzzing::tests::fuzzed_both_arguments (fuzzer runs = 256)
         [PASS] fuzzing::tests::passing
-        [FAIL] fuzzing::tests::failing_fuzz (fuzzer runs = 1, arguments = [2176913491924129583795008547758153887527303407267805286823327310126940911830, 2350787482052932408706155311404918127641790603096372587257662840043562009418])
+        [FAIL] fuzzing::tests::failing_fuzz (fuzzer runs = 1, arguments = [[..], [..]])
```

Comment:
Are those args printed?

Reply:
Yup, actually I can use the exact values (they changed since we use `get_random` more times before, but the seed is still fixed)